tests/index.spec.ts

```diff
@@ -10,6 +10,21 @@ describe("Phone utils", () => {
     });
   });
 
+  describe("Should Return Valid for the following", () => {
+    const validPhones = [
+      "+(250) 785-323-292",
+      "+256 785 323 292",
```

Comment:
This with `+256` should not pass

Reply:
Actually that is a bug, but this PR just is for solving the badly formatted failing issue

Reply:
So should you create a separate issue to address that or I fix it on this PR?

Reply:
Let's have a separate issue and a PR for that.

Reply:
I mixed it with this one when you delayed to answer me.